vuln-remediation.md

```diff
@@ -0,0 +1,105 @@
+
+   1. **Coordinator notifies the Reporter that the patch has been released.** At this point the patch is available as an update to ROS users.
+
+   1. **Disclose the patch to a broader audience as desired.** Follow the vulnerabilty disclosure plan if one exists. Consider notifying the ROS community of the patch availability; for example, with a general post to [ROS discourse](https://discourse.ros.org/).
+
+   1. **Coordinator updates CVE details.** Include a link to the patch and final CVSS details. 
+
+
+## Additional Guidance
+
+### Identifying the Maintainer
+
+If the Maintainer of the vulnerable package is not well known, check for a <maintainer> tag in the package's `package.xml` file. Also review recent activity for the package, and contact a recent pull request approver(s) to find a responsible person.
+
+### Requesting information from the Reporter
+
+Reporters may have a wealth of additional details about the vulnerability which they are willing to share when asked.  Consider requesting some or all of the following:
+
+ - Operating system
+ - ROS distro
+ - ROS package
+ - Robot that you were testing
+ - How to reproduce the issue
+ - Do you have / can you provide a git patch to fix the issue?
+ - Do you have a docker image, a snap, or similar artifacts to share?
+
+### Unresponsive Maintainers
+
+As documented in [REP 2004, Package Quality Categories](https://ros.org/reps/rep-2004.html), ROS packages at quality level 1 and 2 are required to follow disclosure guidelines. The Coordinator should remind the Maintainer of their obligations under REP 2004, and escalate to the ROS 2 [Technical Steering Committee](https://index.ros.org/doc/ros2/Governance/) if necessary.
+
+
+### The `security@openrobotics.org` distribution list
+
+This email distribution list is to be used exclusively for the public to report ROS vulnerabilities. Unless explicitly labeled otherwise, conversations to the list are considered private and confidential. Follow the [CISA Traffic Light Protocol (TLP)](https://www.cisa.gov/tlp) guidance for TLP:RED information usage and sharing.
+
+Distribution list membership will be maintained by Open Robotics. Membership must be broad enough to ensure coverage for timely response to Reporters. No email aliases or shared mailboxes may be included in the list.
+
+Individuals added to the distribution list must adhere to the following principles:
+ - You must be able to take ownership of reported vulnerabilities and act as the Coordinator through remediation. When high risk vulnerabilities are reported, this means vulnerability coordination becomes your highest priority until a fix is released.
+ - You must reach out to the Security Working Group or individual members of the Working group when you need assistance in handling vulnerability reports.
```

Comment:
RE: use single wording for referring to the working group

We seem to be conflating the coordinators and the working group. Currently channels to contact the working group are public and meant to be public. How would one coordinator go through contacting the working group with vulnerability details? what is the responsibility of the working group in these cases ? (non-disclosure, obligation to follow through the remediation etc)

Reply:
Based on our WG discussions, this is written so the coordinator plays an operational role and the working group plays a purely advisory role. The coordinator may not need help with security-specific tasks, so we should be there to advise / support. That could include things like:

 - Whether a CVE is warranted
 - How to request a CVE number
 - Help with CVSS / risk scoring
 - Reaching out to a third party to help with vuln coordination

No special communications channel exists for the Security WG for vuln coordination, so this would come in on one of our [public channels](https://github.com/ros-security/community#communication-channels), but would fork to a private / encrypted conversation to discuss embargoed topics.